### ISSUE TITLE
Salvage has powercrept far out of control and needs a hard reign in.

### ISSUE THREAD

**Issue by moonheart08** [space-wizards/space-station-14#28251]
## The bait/"What the hell" list
- Salvage currently has a common weapon loot (WeaponCrusherDagger/the crusher dagger) which is a hard upgrade over **the captain's sabre** at **30 slashing dps**. (And this is just an example.)
- The majority of salvage mobs are *completely unmatched* by station crew, with no damage sources that can counter them beyond hard armory weapons.
- Salvage can quite literally pack their bags and fuck off, a design issue we've tried to avoid (and gave up on?) since the beginning which is now extremely viable due to the non-existent barriers against doing so.
- Salvage can trivially just buy other department's equipment (and smash the locks) so they don't have to ever talk with them, making their relationship with departments like chemistry moot and removing their incentive to share with them.
- Salvage has the fastest non-GM roundstart hardsuit in the game.
- Salvage has the **best** non-GM hardsuit in the game. (ClothingOuterHardsuitMaxim)
- Expeditions have almost lavaland-tier mobs now, and share many of the core design issues with lavaland with only a very lenient timer to prevent it from becoming an even bigger issue. (The timer has no effect on the primary issues, as they have a portable, independent base in the form of their shuttle at this time.)
## The fix

I have no idea. I'm putting up this issue because this needs properly discussed, and I want to try to formulate a plan to completely gut the current system and steer it back into a saner direction (likely by bringing procedural asteroids and the salvage magnet, their *actual* main mechanics, back into relevance.)

Good changes likely include:
- Upsizing the magnet and ore processor into larger, bulk machines. They should require more power to run, and have other negatives that discourage moving them onto mobile vessels (potentially make them impose an acceleration penalty to thrusters?)
- Reworking all damage and health numbers, possibly starting with just dividing them by some factor, to bring them back in line with station numbers. As a baseline, 22.5 slashing DPS is the Captain's Sabre damage values, and roundstart weapons shouldn't be higher than that.
- I have no clue what to do with expeditions don't ask (or rather, I don't have an idea people here would like, because I want to just remove them entirely and focus on the original main mechanics instead of the usurper.)
- Implement *something* to discourage just smashing open locked crates, possibly a variety of things (different manufacturers having different countermeasures?)
   - Honestly locked crates being noisy in comms and having a chance to be trapped such that they explode or otherwise do something violent on smashing would be sufficient.
   - Possibly make locks alert level aware, or otherwise give depts a way to "authorize" simply unlocking crates with their access without permission (essentially a bandaid to keep nukies flowing nicely, by letting security authorize unlocking gun crates for normal personnel.)
- Sit down and revamp asteroid/debris generation. It has been an example placeholder I wrote to demonstrate how to use the system this entire time, and is effectively just entirely unconfigured as a result. The system is good, but it needs an actual design pass to work for the game.

**Comment by K-Dynamic** [comment]
Most of these issues can be immediately fixed by deleting salvage reclaimer (and rolebanning any salvagers that try to borrow cargo shuttle) and providing mini-jetpacks, as well as moving towards gateways and vg roid

Crusher weapons are kinda strong in general but the crusher dagger is loot-only - I personally preferred the old stats and using it as a permanent lamp though

I personally think salvagers should be split between shaft miners (cargo role) and expedition teams that can select anyone on the station instead of just salvage, giving everyone a chance of getting salv loot

**Comment by moonheart08** [comment]
> Most of these issues can be immediately fixed by deleting salvage reclaimer (and rolebanning any salvagers that try to borrow cargo shuttle) and providing mini-jetpacks, as well as moving towards gateways and vg roid
> 
> Crusher weapons are kinda strong in general but the crusher dagger is loot-only - I personally preferred the old stats and using it as a permanent lamp though
> 
> I personally think salvagers should be split between shaft miners (cargo role) and expedition teams that can select anyone on the station instead of just salvage, giving everyone a chance of getting salv loot


Gateways and vg roid are godawful what are you on about. They **also** have the same issues we've been trying hard to avoid with not having lavaland.

We put in **all of this design work** specifically to **not** have ss13 shaft miners because **they're a terribly designed role through and through**, please do not fall back to just copying them.

**Comment by UbaserB** [comment]
what are you talking about, sloth is actively working on vg roid iirc

**Comment by moonheart08** [comment]
> what are you talking about, sloth is actively working on vg roid iirc
I'm just going to not comment on this, because ***WHY.***

**Comment by moonheart08** [comment]
We spent *ages* designing mechanics like the salvage magnet not only to **completely fail to implement them in full** but apparently also to **completely discard them in favor of shit we said we wouldn't do because it's fucking awful**?
This is news to me and I'm not very pleased about it.

**Comment by moonheart08** [comment]
I suppose it's time to go over why neither gateways nor vg roid are going to help (they're going to hurt), give me a bit.

**Comment by Mith-randalf** [comment]
I can confirm that it defo was the maint position that off-station stuff would be basically nonexistant so as to keep the gameplay focused. I was kinda surprised to see the random shuttle events and pseudo-lavaland stuff. I'm kinda split as to whether it'd be a good thing overall to spread it out, but I just wanted to agree that yes this is categorically a change in design goal imo.

**Comment by Savsj** [comment]
Why not make the reclaimer use actual fuel that only engi can make? If you want to force them back to the station adding a fuel cost is the best way to do it, i personally see no reason to do anything with the magnet except maybe increase its pool of choices to make it more middling, if need be, and ive never seen someone actually use the crusher dagger since it was(rightfully) removed from the borg loadout, i cant speak to the maintainers positions only as someone who plays a lot of cargo. i will also note that in my many many many expeditions and hours in cargo i have literally never seen the maxim suit spawn, I have only ever seen it in admin events. Can anyone confirm it can actually spawn on expeds? I am of the opinion we shoudnt be basing such changes regardless on a <1% chance spawn items unless its truly unhealthy for it to be in the game like the throngler.

**Comment by Agustron24** [comment]
Add a full new job, shaft miners.

Their way to get materials is trough a mounted teleporter to a planet that doesnt have a timer and does not spawn dungeons.

The process to refine ores would be more complex, factorio like, since having a job that is just salvage mining wouldnt be that fun.

Salvage problem would stay unsolved but atleast we got materials now.

**Comment by Agustron24** [comment]
Soo salvage: Go get scrap and loot from wrecks (lethal company)

Miners: go get materials for the station (factorio)

**Comment by Agustron24** [comment]
Make salvage only find scrapped versions of those things, and then they could be researched at a tech disk scanner, that will scan the item and give a techdisk from the item.

Basically like grabbing left behind tech from the enemy and then using it against them.

You also can scan regular syndie items for also getting their disk, soo any syndie tech that is stolen the station will get it for themselves.


This would make salvage more dependant for the station since you can make your whole team have blood red magboots wich are essentially jetpacks.

Also making these would require mats soo go mine.


Probably OP but this shit will be expensive to make and valuable to sell soo pretty much you have to mine for getting it.

**Comment by moonheart08** [comment]
have you considered that you don't actually need gamer loot to have a game loop

**Comment by GroggleG** [comment]
Realistically there is no quick fix nor is it desirable for me at least. We need a big redesign for it not be a giant problem or a unplayable and not fun role, either Ill go make a redesign doc or someone else is as a shaft miner main on TG/Goon.

**Comment by ninruB** [comment]
Salvage is redundant for SS gameplay in general because it is by design a form of external money generation for cargo that barely interacts with the rest of the station, by design.

This means that if an overarching design goal of player interaction and system intertwinement is pursued salvage is actively harmful because it introduces a closed-loop gameplay pattern and system.

Now we are at a crossroad, we either remove salvage and replace it with in-station mechanics along with some form of passive external money generation, or we fix salvage.

Salvage can be fixed if its woven into the rest of the gameplay loop at its core. Salvage gets materials and everyone else wants materials, so everyone else should be working with salvage to get their materials.

Ideaguying further I'll propose an analogy: a mineshaft.
The salvage crew are the miners, they dig the mineshaft and extract resources.

But digging the mineshaft and extracting resources is dangerous because the mineshaft is less stable and moving ores without infrastructure is tedious and inefficient, so engineers come in.
But digging the mineshaft can reveal harmful gasses, and air might be difficult to reach into the mineshaft, so atmos techs are required.

Since we're a silly videogame we can have monsters in mineshafts and striving for more interactivity we can also add workplace hazards like pitfalls, collapses and explosions, which leads to harm.

Here we can either add this onto security responsibilities, thus tying them in as well, or delegate this to salvage giving them more autonomy. I think that security should be tied in to highlight that the mineshaft is PART of the station, just an external one, not some lawless frontier.

Harm leads to woe and death which may harm anyone interacting with the mineshaft, so doctors are required, but they need to be supplied, so chemistry and botany are tied in.

But digging is hard work and you'll need energy for that, energy in the context of people is acquired through food, so botany and the chef are tied in.

Mineshafts can lead to mysterious areas and anomalous occurrences, so you can tie science in so they come and research it for points. Science is already naturally tied in because, you know, upgrades, but having them directly interact is much better.

Cargo is naturally receiving the resources from salvage and distributing or liquidating them, allowing the crew to receive its share of materials and allowing cargo to function as a department.

Instead of having the mineshaft be on some far away mining planet or connected to some arbitrary place with a teleporter I propose an infinitely more based solution of a giant fuckoff station-planet elevator or tractor beam because its SILLY and WONDROUS.

This is another "hey redesign mining/salvage/whatever from scratch including core gameplay loop restructing" kind of salvage ideaguying but if anything its probably better than what we have right now by a lot.

**Comment by GroggleG** [comment]
> Salvage is redundant for SS gameplay in general because it is by design a form of external money generation for cargo that barely interacts with the rest of the station, by design.
> 
> This means that if an overarching design goal of player interaction and system intertwinement is pursued salvage is actively harmful because it introduces a closed-loop gameplay pattern and system.
> 
> Now we are at a crossroad, we either remove salvage and replace it with in-station mechanics along with some form of passive external money generation, or we fix salvage.
> 
> Salvage can be fixed if its woven into the rest of the gameplay loop at its core. Salvage gets materials and everyone else wants materials, so everyone else should be working with salvage to get their materials.
> 
> Ideaguying further I'll propose an analogy: a mineshaft. The salvage crew are the miners, they dig the mineshaft and extract resources.
> 
> But digging the mineshaft and extracting resources is dangerous because the mineshaft is less stable and moving ores without infrastructure is tedious and inefficient, so engineers come in. But digging the mineshaft can reveal harmful gasses, and air might be difficult to reach into the mineshaft, so atmos techs are required.
> 
> Since we're a silly videogame we can have monsters in mineshafts and striving for more interactivity we can also add workplace hazards like pitfalls, collapses and explosions, which leads to harm.
> 
> Here we can either add this onto security responsibilities, thus tying them in as well, or delegate this to salvage giving them more autonomy. I think that security should be tied in to highlight that the mineshaft is PART of the station, just an external one, not some lawless frontier.
> 
> Harm leads to woe and death which may harm anyone interacting with the mineshaft, so doctors are required, but they need to be supplied, so chemistry and botany are tied in.
> 
> But digging is hard work and you'll need energy for that, energy in the context of people is acquired through food, so botany and the chef are tied in.
> 
> Mineshafts can lead to mysterious areas and anomalous occurrences, so you can tie science in so they come and research it for points. Science is already naturally tied in because, you know, upgrades, but having them directly interact is much better.
> 
> Cargo is naturally receiving the resources from salvage and distributing or liquidating them, allowing the crew to receive its share of materials and allowing cargo to function as a department.
> 
> Instead of having the mineshaft be on some far away mining planet or connected to some arbitrary place with a teleporter I propose an infinitely more based solution of a giant fuckoff station-planet elevator or tractor beam because its SILLY and WONDROUS.
> 
> This is another "hey redesign mining/salvage/whatever from scratch including core gameplay loop restructing" kind of salvage ideaguying but if anything its probably better than what we have right now by a lot.

This is literally shaft miner but not shaft miner, just TWEAK salvage to be more about mining and endgame coolfactor shit like lavaland loot (I know that gameplay loop isnt wanted but what do we do to MAKE people WANT to play a mining role? artefacts and trinkets thats how)

**Comment by Agustron24** [comment]
Make a factorio like refining sistem for ores

**Comment by Erisfiregamer1** [comment]
I'm going to try to go through the points as a regular player of Salvage:

> Salvage currently has a common weapon loot (WeaponCrusherDagger/the crusher dagger) which is a hard upgrade over the captain's sabre at 30 slashing dps. (And this is just an example.)

That is broken, I agree. A PKA is usually sufficient. (Why do we have crushers anyway?)

> The majority of salvage mobs are completely unmatched by station crew, with no damage sources that can counter them beyond hard armory weapons.

Why would salvage mobs be on the station, apart from carps from a dragon? You only ever find them on magnet wrecks or expeditions, and on both you'd usually have a PKA to deal with them.

I do agree that maybe nerfing them a bit if they *do* get on station would be nice though.

> Salvage can quite literally pack their bags and fuck off, a design issue we've tried to avoid (and gave up on?) since the beginning which is now extremely viable due to the non-existent barriers against doing so.

This is bad, I agree. Maybe make it so that they need to get fuel like Tritium from the station for flying the ship. (Correct me if I'm wrong but you can't buy Tritium) [UnityStation does something similar, afaik you need to fuel thrusters to make them work]
At the same time, incentivizing coming back would help a lot. Giving Science things Salvage could *really* use, while also preventing Salvage from getting them from off-station would work wonders. (Right now the closest thing available is weapons and advanced tools, which isn't really needed for expeditions or wrecks)

> Salvage can trivially just buy other department's equipment (and smash the locks) so they don't have to ever talk with them, making their relationship with departments like chemistry moot and removing their incentive to share with them.

Make the crates impossible to break until it's unlocked, or just tamper-resistant in a way that destroying them deletes the contents.

> Salvage has the fastest non-GM roundstart hardsuit in the game.

If you're referring to the Spationaut suit, it's really not that strong. If you're referring to the mining hardsuit, that's unobtainable except from wrecks. It does need a nerf though.

> Salvage has the best non-GM hardsuit in the game. (ClothingOuterHardsuitMaxim)

Wait, when was Maxim obtainable? I've never seen it gotten in a single round.

> Expeditions have almost lavaland-tier mobs now, and share many of the core design issues with lavaland with only a very lenient timer to prevent it from becoming an even bigger issue.

Can someone explain why this is an issue? Even from a maintainer perspective, these mobs aren't hard to kill (except Space Dragons and Xeno Queens, but those aren't really that hard, a team of 3 salvs could take them down)

My apologies for the textwall, this is just me trying to help address this. I agree Salvage is way too strong. I agree they should be nerfed.

**Comment by Kayek14** [comment]
> what do we do to MAKE people WANT to play a mining role? artefacts and trinkets thats how)

Why do we need to add any gamerloot to the mining role? Janitors dont get gamerloot, Chemists dont get gamerloot. Miners/salvage should be like any other role. Will people only be playing miner/salvage just for the gamerloot? I dont think we should be dangeling a reward to get people to do the job. Honestly i would like to play miner just to relax and mine ores for the station. I dont need no silly loot. 

Adding trinkets and artifacts leads to people just playing the role for the reward, and not for the auctual job and the benifit it brings to the station.

**Comment by HolySSSS** [comment]
Good bait.

Remove the Reclaimer. 
Let the jetpacks do the work.
Engineers will have more reason to make shuttles.

Make expeditions last 3.5±0.5 minutes, add a minimum though.
Play with the variables, I myself recommend giving more randomness to the timer.
Add a countdown timer (as a item | UI) for fairness, a good salvager will know when to leave.

Minimuse loot and make it more specific, the salvagers must pay for access to a disk.
The expedition console will provide information about what kind of a base is down there.
Salvagers will have to consider the worth of a expedition then with QM and not go cause the cooldown went down.

Have fun brainstorming further.

**Comment by GroggleG** [comment]
> Good bait.
> 
> Remove the Reclaimer. Let the jetpacks do the work. Engineers will have more reason to make shuttles.
> 
> Make expeditions last 3.5±0.5 minutes, add a minimum though. Play with the variables, I myself recommend giving more randomness to the timer. Add a countdown timer (as a item | UI) for fairness, a good salvager will know when to leave.
> 
> Minimuse loot and make it more specific, the salvagers must pay for access to a disk. The expedition console will provide information about what kind of a base is down there. Salvagers will have to consider the worth of a expedition then with QM and not go cause the cooldown went down.
> 
> Have fun brainstorming further.

Short ass expeds, guh? Expeds shouldnt have a time limit and should just be on an FTL basis. Kinda agree with the Reclaimer but if we WERE to do that make tinyfans craftable ffs. Make ACTUALLY FUN LOOT rarer and have to work for it, add exped tiers like intended (lavaland and such ((never)) and make salvage more reliant on the station

**Comment by moonheart08** [comment]
> I'm going to try to go through the points as a regular player of Salvage:
> 
> > Salvage currently has a common weapon loot (WeaponCrusherDagger/the crusher dagger) which is a hard upgrade over the captain's sabre at 30 slashing dps. (And this is just an example.)
> 
> That is broken, I agree. A PKA is usually sufficient. (Why do we have crushers anyway?)
> 
> > The majority of salvage mobs are completely unmatched by station crew, with no damage sources that can counter them beyond hard armory weapons.
> 
> Why would salvage mobs be on the station, apart from carps from a dragon? You only ever find them on magnet wrecks or expeditions, and on both you'd usually have a PKA to deal with them.
> 
> I do agree that maybe nerfing them a bit if they _do_ get on station would be nice though.
> 
> > Salvage can quite literally pack their bags and fuck off, a design issue we've tried to avoid (and gave up on?) since the beginning which is now extremely viable due to the non-existent barriers against doing so.
> 
> This is bad, I agree. Maybe make it so that they need to get fuel like Tritium from the station for flying the ship. (Correct me if I'm wrong but you can't buy Tritium) [UnityStation does something similar, afaik you need to fuel thrusters to make them work] At the same time, incentivizing coming back would help a lot. Giving Science things Salvage could _really_ use, while also preventing Salvage from getting them from off-station would work wonders. (Right now the closest thing available is weapons and advanced tools, which isn't really needed for expeditions or wrecks)
> 
> > Salvage can trivially just buy other department's equipment (and smash the locks) so they don't have to ever talk with them, making their relationship with departments like chemistry moot and removing their incentive to share with them.
> 
> Make the crates impossible to break until it's unlocked, or just tamper-resistant in a way that destroying them deletes the contents.
> 
> > Salvage has the fastest non-GM roundstart hardsuit in the game.
> 
> If you're referring to the Spationaut suit, it's really not that strong. If you're referring to the mining hardsuit, that's unobtainable except from wrecks. It does need a nerf though.
> 
> > Salvage has the best non-GM hardsuit in the game. (ClothingOuterHardsuitMaxim)
> 
> Wait, when was Maxim obtainable? I've never seen it gotten in a single round.
> 
> > Expeditions have almost lavaland-tier mobs now, and share many of the core design issues with lavaland with only a very lenient timer to prevent it from becoming an even bigger issue.
> 
> Can someone explain why this is an issue? Even from a maintainer perspective, these mobs aren't hard to kill (except Space Dragons and Xeno Queens, but those aren't really that hard, a team of 3 salvs could take them down)
> 
> My apologies for the textwall, this is just me trying to help address this. I agree Salvage is way too strong. I agree they should be nerfed.

in short to *all* of this: A gameplay tenant we upheld (I'm not sure if we do anymore) was to avoid placing crew too far from the station, because lavaland gameplay is fun for exactly *one* group of people or person, and detrimental to everyone else for a small variety of reasons. That's why I'm baffled we suddenly went all out on offstation-heavy gameplay, because we agreed a while ago it was a terrible idea and if/*when* mobs and items from those environments leak over to the station they throw everything out of balance on the spot due to just being better than crew equipment by default.

**Comment by Erisfiregamer1** [comment]
> in short to _all_ of this: A gameplay tenant we upheld (I'm not sure if we do anymore) was to avoid placing crew too far from the station, because lavaland gameplay is fun for exactly _one_ group of people or person, and detrimental to everyone else for a small variety of reasons. That's why I'm baffled we suddenly went all out on offstation-heavy gameplay, because we agreed a while ago it was a terrible idea and if/_when_ mobs and items from those environments leak over to the station they throw everything out of balance on the spot due to just being better than crew equipment by default.

I get what you mean. However, as a salvmain, being able to do something not related to the *eternal painful grind* of the station and being able to *explore* is fun. It's why I play the role.

Maybe make it so people get times where they can go off the station too and explore. Maybe find some loot on those minor expeditions themselves, instead of just... not letting crew leave the station?

**Comment by metalgearsloth** [comment]
tl;dr this belongs on the feedback forum to bikeshed not github.

To address some points raised:
- Reclaimer is getting removed at some point and shuttle building is getting made easier.
- IDK what a maxim suit is but might need looking at.
- Doc will be made at some point.

**Comment by Golinth** [comment]
> IDK what a maxim suit is but might need looking at.

Its a top tier salv hardsuit with some wild stats. (No slowdown, 40%+ all brute resist, 70(!!!)% heat resist, and 90% rad resist.) Its the nukie elite suit but better. That being said, I rarely see it in game and think it might be admeme only atm.

**Comment by Savsj** [comment]
It IS admeme only, and its pretty much sole use is to make the QM feel cool during event rounds

**Comment by GroggleG** [comment]
> > IDK what a maxim suit is but might need looking at.
> 
> Its a top tier salv hardsuit with some wild stats. (No slowdown, 40%+ all brute resist, 70(!!!)% heat resist, and 90% rad resist.) Its the nukie elite suit but better. That being said, I rarely see it in game and think it might be admeme only atm.

Honestly just sounds like a reasonable craftable if it gets nerfed by 20% Require goliath hide or some shit as I advocate for a lavalandlike

**Comment by metalgearsloth** [comment]
> tl;dr this belongs on the feedback forum to bikeshed not github.
